src/tools/scratchtool.c

```diff
@@ -80,13 +80,15 @@ static int do_rados_getxattrs(rados_ioctx_t io_ctx, const char *oid,
 		for (i = 0; i < nval; ++i) {
 			if (strcmp(exkeys[i], key))
 				continue;
-			if ((len == strlen(exvals[i]) + 1) && (val != NULL) && (!strcmp(exvals[i], val))) {
-				nfound++;
-				break;
+			if(val != NULL){
```

Comment:
missing spaces around parentheses

Reply:
looking more closely, I don't see the utility of this fix; val is not used until the val != NULL part of the if(), and if that test fails, the strcmp to exvals[i] will not be performed.  Is it trying to avoid the use of val in the printf()?  Linux printf() handles this, but others may not?...was there an actual bug caused by this code?